packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1977,6 +1977,24 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
             ))! as List<Object?>;
           }
 
+          /// Returns whether the child was created by the local project.
+          bool wasCreatedByLocalProject(Map<String, Object?> childJson) {
+            return childJson['createdByLocalProject'] != null &&
+                childJson['createdByLocalProject'] == true;
+          }
+
+          /// Returns whether the child has a description matching [description].
+          bool hasDescription(Map<String, Object?> childJson, {required String description}) {
+            return childJson['description'] != null &&
+                childJson['description'] == description;
+          }
+
+          /// Returns whether the child has a text preview matching [preview].
+          bool hasTextPreview(Map<String, Object?> childJson, {required String preview}) {
+            return childJson['textPreview'] != null &&
```

Comment:
this can just be `childJson['textPreview'] == preview`. Since `preview` is non-nullable, the equality check will suffice to verify that the value from the json is non-null.

Same for the methods above

Reply:
good point, done!